src/components/AppLayout/NavHeader.tsx

```diff
@@ -134,23 +136,60 @@ const NavHeader = ({ className, sidedrawer, setSidedrawer, displayName, isVerifi
 					href='https://polkasafe.xyz/'
 					target='_blank'
 					rel='noreferrer'
-					className='custom-link'
+					className='custom-link after:hidden'
 				>
-					<span className='flex items-center gap-x-2 text-sm font-medium text-bodyBlue hover:text-pink_primary dark:text-blue-dark-high dark:hover:text-pink-dark-primary'>
-						{theme === 'dark' ? <PolkasafeWhiteIcon className='relative left-[3px] top-[-1px] scale-[2]' /> : <PolkaSafe />}
+					<span className='flex items-center gap-x-2 text-sm font-medium text-bodyBlue hover:text-pink_primary dark:text-blue-dark-high dark:hover:text-pink_primary'>
+						{theme === 'dark' ? <PolkasafeWhiteIcon /> : <PolkaSafe />}
 						<span>Polkasafe</span>
 					</span>
 				</a>
 			)
+		},
+		{
+			className: 'logo-class',
+			key: 'Staking',
+			label: (
+				<a
+					href='https://staking.polkadot.network/'
+					target='_blank'
+					rel='noreferrer'
+					className='custom-link after:hidden'
+				>
+					<span className='flex items-center gap-x-2 text-sm font-medium text-bodyBlue hover:text-pink_primary dark:text-blue-dark-high dark:hover:text-pink_primary'>
+						<StakeIcon />
+						<span>Staking</span>
+					</span>
+				</a>
+			)
 		}
 	];
 
+	if (['kusama', 'polkadot'].includes(network)) {
```

Comment:
create a variable like 
export const delegationSupportedNetworks = [
	AllNetworks.KUSAMA,
	AllNetworks.POLKADOT
];
in Delegation component
and then use that 
also change this in pages/delegation/index.tsx file